reg_tests/grid_gen/driver.hera.sh

```diff
@@ -45,8 +45,8 @@ export APRUN=time
 export APRUN_SFC=srun
 export OMP_STACKSIZE=2048m
 export machine=HERA
-export NCCMP=/scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/v1.0.0-beta1/intel-18.0.5.274/impi-2018.0.4/nccmp/1.8.7.0/bin/nccmp
 export HOMEreg=/scratch1/NCEPDEV/da/George.Gayno/noscrub/reg_tests/grid_gen/baseline_data
+export NCCMP=$(which nccmp)
```

Comment:
Do you need these exports?

Wherever you are calling nccmp, you could put it there as:

NCCMP=${NCCMP:-$(which nccmp)}

You don't need to do `export NCCMP=$(which nccmp)` in any of the driver scripts; e.g. `driver.hera.sh`

Reply:
Done for the Hera driver scripts.  I can update the remaining scripts when I switch to the hpc-stack on all machines (#152)